src/services/midgard/types.ts

```diff
@@ -27,3 +28,10 @@ export type PoolsState = {
 export type PoolsStateRD = RD.RemoteData<Error, PoolsState>
 
 export type NetworkInfoRD = RD.RemoteData<Error, NetworkInfo>
+
+export type PoolCurrency = {
+  symbol: string
+  name: string
+  poolData: PoolData
+}
+export type PoolCurrencies = PoolCurrency
```

Comment:
You can ignore types of `PoolCurrency` + `PoolCurrencies`. These are needed to deal with real data for `HeaderCurrency`, but that will be another PR.